src/components/app-content-switcher/AppContentSwitcher.tsx

```diff
@@ -1,4 +1,4 @@
-import { FC } from 'react'
+import React, { FC } from 'react'
```

Comment:
Do we actually need this import?
```suggestion
import { FC } from 'react'
```